compat_test.go

```diff
@@ -154,7 +154,7 @@ func (*suite) TestSignature(c *gc.C) {
 		c.Logf("test %d: %s", i, test.about)
 		checkConsistency(c, func(pkg mcompat.Package) (interface{}, error) {
 			m := makeMacaroon(pkg, test.macaroon)
-			c.Logf("made macaroon %s with sig %x", macStr(m), m.Signature())
+			//c.Logf("made macaroon %s with sig %x", macStr(m), m.Signature())
```

Comment:
I'm not keen on having commented out code. please remove it if it isn't needed or replace it.